R/dm.R

```diff
@@ -39,12 +41,15 @@
 #' dm_nycflights13() %>% tbl("airports")
 #' dm_nycflights13() %>% src_tbls()
 #' dm_nycflights13() %>% dm_get_src()
+#' # this works only when tables of `dm` are on DB
+#' try(dm_nycflights13() %>% dm_get_con())
```

Comment:
Can you copy to `src_memdb()` and then show the connection? Perhaps in a `if (FALSE) {}` block?